### ISSUE TITLE
Question: Correct interpretation of summary_plot graph

### ISSUE THREAD

**Issue by Sanchita-P** [shap/shap#981]
Hi!

While through the various resources online to understand the shap plots, I ended up slightly confused. Find below my interpretation of the overall plot given in examples - 
![image](https://user-images.githubusercontent.com/47073198/71723446-ad6cb500-2e52-11ea-9cc6-04f8a5b71552.png)
1. Shap value 0 for a feature corresponds to the average prediction using all the other possible combinations of features except for the feature of interest. For e.g, shap value 0 for LSTAT corresponds to the average prediction of models having different combinations of features (excluding LSTAT)
2. Shap value of 4 with respect to 0 means that there is a relative increase in prediction being 1. How do we explain shap values 4, 2, etc. with respect to 0 in layman's terms?
3. Slight confusion in interpreting the overall graph vs the single point graph (
![image](https://user-images.githubusercontent.com/47073198/71723663-7cd94b00-2e53-11ea-92ad-61ac9b8a7600.png)) - 
In a single data point example, the base value/expected value is the average prediction in the entire dataset -> but in the overall plot (referenced at the top) we are saying it is the average prediction without that feature. So things don't add up?

Thank you for your help! @slundberg :)

**Comment by amirrezamafi** [comment]
I`m not sure that I understood your question completely, But my understanding about SHAP (if it be correct :) ) is:
1. If for one instance (x), the SHAP_value for a feature (LSTAT) is 0, it means that based on SHAP, this feature has no significant effect in the output of the model **for this instance`s output**, in respect of all the coalition of the features in the model
2. the SHAP_value = 4 for a feature(f1) does not have specific meaning alone. but in compare of another feature (f2) with SHAP_value = 2 for the same instance, It says that the feature f1 has more effect than feature f2 on the final output of the model 
3. my understanding on the force_plot is:
- the grey values above the axis are the range of predictions.
- the bold black value is the prediction for this instance by our ML model. 
- the base value is the average output for all the dataset
- the red bars illustrate the features with positive SHAP values (push to increase the prediction to class_1). Each red bar represents a feature (LSTAT) and its value (4.98) in the dataset, and the bar`s magnitude represents its SHAP value (feature importance).
- the blue bars illustrate the features with negative SHAP values (push to decrease the prediction to class_0). Each blue bar represents a feature (RM) and its value (6.575)in the dataset, and the bar`s magnitude represents its SHAP value (feature importance). 
@slundberg , Please let us know if my understanding is right. Thank you
